frontend/src/components/ui/RunPipelineButton.jsx

```diff
@@ -75,9 +75,11 @@ export default function RunPipelineButton({modalPopper, children, action}) {
       pipelineSpecs['build'] = pipeline.buffer
       pipelineSpecs['name'] = pipeline.name
       pipelineSpecs['id'] = pipeline.id
+      const rebuild = (action == "Rebuild")
       const execution = {
         id: executionId,
         pipeline: pipelineSpecs,
+        build: rebuild
```

Comment:
So the rebuild button also triggers a run?

Reply:
Yep